pkg/scheduler/apis/config/types.go

```diff
@@ -220,9 +220,20 @@ type PluginConfig struct {
 	Args runtime.Unknown
 }
 
-///////////////////////////////////////////////////////////////////////////////
-// NOTE: The following methods are intentionally left out of the staging mirror.
-///////////////////////////////////////////////////////////////////////////////
+/*
+ * NOTE: The following variables and methods are intentionally left out of the staging mirror.
+ */
+const (
+	// DefaultPercentageOfNodesToScore defines the percentage of nodes of all nodes
+	// that once found feasible, the scheduler stops looking for more nodes.
+	DefaultPercentageOfNodesToScore = 50
+
+	// MaxCustomPriorityScore is the max score UtilizationShapePoint expects.
+	MaxCustomPriorityScore int64 = 10
+
+	// MaxWeight defines the max weight value allowed for custom PriorityPolicy
+	MaxWeight = 1000000
```

Comment:
this does not match previous behavior (see question at original location)

Reply:
The primary motivation of not specifying it as `MaxWeight int64 = MaxTotalScore / MaxCustomPriorityScore ` is to avoid dependency on pkg `pkg/scheduler/framework/...`.

I agree we should be consistent. How about duplicating `MaxTotalScore` here (if needed in the future, should update the value in both `pkg/scheduler/framework/...` and here).